src/dataDisplay/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,50 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap");
+
+.avatarWrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Inter", sans-serif;
+  height: 40px;
+  width: 40px;
+}
+
+.templateAvatar {
+  border-radius: 50%;
+}
+
+.templateDefault {
+  position: relative;
+  border-radius: 50%;
+  background-color: #23A094;
+}
+
+.templateDefault p {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #FFFFFF;
+  margin: 0;
+}
+
+.avatarBig {
+  max-width: 40px;
```

Comment:
**nitpick:** This element is so small that you can easily set it to fixed `width`, and `height`. No need to set maxWidth, and maxHeight